apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTest.h

```diff
@@ -335,12 +347,14 @@ class AudioWorkloadTest : oboe::AudioStreamDataCallback {
             if (mPhase >= M_PI) mPhase = mPhase - 2.0f * M_PI;
         }
 
-        mSynthWorkload.onCallback(currentVoices);
-        if (currentVoices > 0) {
-            // Render synth workload into the buffer or discard the synth voices.
-            float *buffer = (audioStream->getChannelCount() == 2 && mHearWorkload)
-                            ? static_cast<float *>(audioData) : nullptr;
-            mSynthWorkload.renderStereo(buffer, numFrames);
+        if (mSynthWorkload) {
```

Comment:
If mSynthWorkload is null, does it affect the test result? If so, it may be good to abort the test earlier if mSynthWorkload is null or return error immediately when opening and close the stream.

Reply:
I thinking leaving this alone is fine for now